### PR TITLE
when strictSsl is false set GIT_SSL_NO_VERIFY=true for git command

### PR DESCRIPTION
**Pull request by pwielgolaski**
Solves #2129

### CODE REVIEW

**Comment by pwielgolaski** [comment]
it seems to failing on travis, but it is not related to the change

**Comment by faceleg** [comment]
I've restarted the build, let's cross our fingers.

**Comment by faceleg** [comment]
The board is green!

**Comment by faceleg** [comment]
OK I'll try to merge this in later today!

File: lib/core/resolvers/GitResolver.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ function GitResolver(decEndpoint, config, logger) {
     // anyway
     mkdirp.sync(config.storage.empty);
     process.env.GIT_TEMPLATE_DIR = config.storage.empty;
+    if (!config.strictSsl) {
+        process.env.GIT_SSL_NO_VERIFY = 'true';
+    }
```
Comment (sheerun) [inline lib/core/resolvers/GitResolver.js]: I think we need "else" part as well to support api interface.. Currently in API interface it's possible to run multiple bower commands in one process.

Comment (pwielgolaski) [inline lib/core/resolvers/GitResolver.js]: Sorry, I dont get what should be in "else" part.
How is it different from  ``process.env.GIT_TEMPLATE_DIR`` line above?

Comment (sheerun) [inline lib/core/resolvers/GitResolver.js]: GIT_TEMPLATE_DIR is set every time (if config.storage.empty is false or true), and GIT_SSL_NO_VERIFY only if config.strictSsl is false. You don't handle the case when config.strictSsl is true.

**Comment by pwielgolaski** [comment]
What do you say now, can it go into master?

**Comment by sheerun** [comment]
No, because you still handle only one case. You need to drop if, of add else.

**Comment by pwielgolaski** [comment]
Let me explain my motivation behind 'if', I think that if someone has already defined explictly the variable, we should not change it and respect it.
If you disagree, I can remove the condition.

**Comment by sheerun** [comment]
We should be consistent, so just like in GIT_TEMPLATE_DIR case, we'll force the value.

**Comment by pwielgolaski** [comment]
Ok, I changed it as you proposed.

**Pull request merged**